packages/wonder-blocks-layout/components/media-layout.js

```diff
@@ -117,9 +120,11 @@ class MediaLayoutInternal extends React.Component<CombinedProps, State> {
         if (this.state.size) {
             return this.state.size;
         } else {
-            for (const [size, spec] of ((Object.entries(
-                this.props.mediaSpec,
-            ): any): Array<[MediaSize, {query: string}]>)) {
+            const entries = ((Object.entries(this.props.mediaSpec): any): Array<
+                [MediaSize, {query: string}],
+            >);
```

Comment:
NIT: I think it's better to define the type of `entries` as this is setting an expectation rather than allowing it to be inferred from the assignment. (i.e. move the type off the end of the second cast and use it to type the variable being assigned.